modules/EvseManager/Charger.hpp

```diff
@@ -178,8 +173,8 @@ class Charger {
 
     EvseState getCurrentState();
     sigslot::signal<EvseState> signalState;
-    sigslot::signal<types::evse_manager::ErrorEnum> signalError;
-    // /Deprecated
+    // sigslot::signal<types::evse_manager::ErrorEnum> signalError;
```

Comment:
Could this be removed entirely now?

Reply:
removed

Reply:
The rest of the "deprecated" things (EvseState) is still used in other places in EvseManager